sparql11/data-sparql11/protocol/manifest.ttl

```diff
@@ -1,961 +1,1608 @@
+        ht:httpVersion "1.1";
+        ht:methodName "GET";
+        ht:resp [
+          a ht:Response;
+          ht:statusCodeValue "4XX"
+        ]
+      ])
+  ];
+  mf:name "invoke query operation with more than one query string" .
+
+:bad_multiple_updates a mf:ProtocolTest;
+  dawgt:approval dawgt:Approved;
+  dawgt:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql?using-graph-uri=http%3A%2F%2Fkasei.us%2F2009%2F09%2Fsparql%2Fdata%2Fdata0.rdf";
+        ht:body [
+          a cnt:ContentAsText;
+          cnt:characterEncoding "UTF-8";
+          cnt:chars "update=CLEAR%20ALL&update=CLEAR%20DEFAULT"
+        ];
+        ht:headers ([
+            a ht:RequestHeader;
+            ht:fieldName "Content-Type";
+            ht:fieldValue "application/x-www-form-urlencoded";
+            ht:headerElements ([
+                a ht:HeaderElement;
+                ht:elementName "application/x-www-form-urlencoded"
+              ])
+          ]);
+        ht:httpVersion "1.1";
+        ht:methodName "POST";
+        ht:resp [
+          a ht:Response;
+          ht:statusCodeValue "4XX"
+        ]
+      ])
+  ];
+  mf:name "invoke update operation with more than one update string" .
+
+:bad_query_method a mf:ProtocolTest;
+  dawgt:approval dawgt:Approved;
+  dawgt:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql?query=ASK%20%7B%7D&default-graph-uri=http%3A%2F%2Fkasei.us%2F2009%2F09%2Fsparql%2Fdata%2Fdata0.rdf";
+        ht:httpVersion "1.1";
+        ht:methodName "PUT";
+        ht:resp [
+          a ht:Response;
+          ht:statusCodeValue "4XX"
+        ]
+      ])
+  ];
+  mf:name "invoke query operation with a method other than GET or POST" .
+
+:bad_query_missing_direct_type a mf:ProtocolTest;
+  dawgt:approval dawgt:Approved;
+  dawgt:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql?default-graph-uri=http%3A%2F%2Fkasei.us%2F2009%2F09%2Fsparql%2Fdata%2Fdata0.rdf";
+        ht:body [
+          a cnt:ContentAsText;
+          cnt:characterEncoding "UTF-8";
+          cnt:chars "ASK {}"
+        ];
+        ht:headers ([
+            a ht:RequestHeader;
+            ht:fieldName "Content-Type";
+            ht:fieldValue "application/x-www-form-urlencoded";
+            ht:headerElements ([
+                a ht:HeaderElement;
+                ht:elementName "application/x-www-form-urlencoded"
+              ])
+          ]);
```

Comment:
I don't think the original had a value set for the `Content-Type` header.

Reply:
You're correct. Code was inferring this for POST requests without an explicit Content-Type. Fixed in 515c519abf9c5615862ad559c0fc35eb71b52759.